src/effects/defs.h

```diff
@@ -22,8 +22,10 @@ enum class SignalProcessingStage {
     Postfader
 };
 
-inline uint qHash(SignalProcessingStage stage) {
-    return static_cast<uint>(stage);
+inline uint qHash(
+        SignalProcessingStage stage,
+        uint seed = 0) {
```

Comment:
What is the need for the default value for `seed`?

Reply:
This is the exact definition used and recommended by Qt:
https://doc.qt.io/qt-5/qhash.html#qhash

> ...(note that you can simply define a two-arguments version, and use a default value for the seed parameter)